flink-libraries/flink-python/src/main/java/org/apache/flink/python/api/PythonOperationInfo.java

```diff
@@ -92,6 +96,11 @@ public PythonOperationInfo(PythonPlanStreamer streamer) throws IOException {
 			values[x] = streamer.getRecord();
 		}
 		parallelism = (Integer) streamer.getRecord(true);
+
+		envID = environmentID;
+		uniqueID = "" + envID + "." + setID;
```

Comment:
All these unique ID's appear unnecessary. Could we not simply clear the `sets` `HashMap` in the PythonPlanBinder when an execution is finished?